_posts/addons/2017-10-23-tcp-addon.md

```diff
@@ -96,6 +110,13 @@ A limit burst is granted for every container. This burst is equal to 10 times
 the limit, so if you have a limit of 10 packets/s, your burst will be set to 100
 packets.
 
+## Limitations
+
+To be able to load balance tcp connections, connections are proxied to your
```

Comment:
-> "Thus, the device identification should be part of the protocole you designed over the TCP connection"?